server.go

```diff
@@ -10,6 +10,8 @@ import (
 	"os"
 	"strings"
 	"time"
+	"flag"
```

Comment:
Nit: Should be alpha sorted

(`gometalinter` will complain the next time I touch the file 😜 )